package.json

```diff
@@ -244,6 +252,11 @@
                 "title": "Create Cluster",
                 "category": "Kubernetes"
             },
+            {
+                "command": "extension.vsKubernetesUseContext",
+                "title": "Set to current cluster",
```

Comment:
Is this consistent with the capitalisation of other commands?

Reply:
Nice catch.